pkgs/applications/networking/python-eduvpn-client/default.nix

```diff
@@ -0,0 +1,45 @@
+{ pkgs, stdenv, fetchFromGitHub, pythonPackages, gtk3, libnotify
+, gobjectIntrospection, networkmanager_openvpn, wrapGAppsHook
+, gdk_pixbuf }:
+
+with pythonPackages;
+
+buildPythonApplication rec {
+  pname = "eduvpn-client";
+  version = "1.0rc11";
+
+  src = fetchFromGitHub {
+    owner = "eduvpn";
+    repo = "python-${pname}";
+    rev = "${version}";
+    sha256 = "1lw6ip2d511ppfl40bfnqadbmjw73ml5m2fabrv6h778icv29nkg";
+  };
+
+  prePatch = ''
+    substituteInPlace eduvpn/util.py \
+      --replace "/usr/local" "$out"
+  '';
+
+  checkInputs = [
+    pytest mock
+  ];
+
+  nativeBuildInputs = [ wrapGAppsHook pytestrunner ];
+
+  propagatedBuildInputs = [
+    gtk3 libnotify dbus-python pygobject3 pynacl configparser
+    requests requests_oauthlib future mock repoze_lru
+    dateutil gobjectIntrospection networkmanager_openvpn six
+    gdk_pixbuf
+  ];
+
+  # Checks need X environment
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/eduvpn/python-eduvpn-client";
```

Comment:
`inherit (src.meta) homepage`

Reply:
Added the overall homepage of the eduVPN project, as that gives more context.